src/probes/isLiteral.js

```diff
@@ -37,6 +37,9 @@ function main(node, options) {
       analysis.addWarning("encoded-literal", node.value, node.loc);
     }
   }
+  else if (/(http[s]?:\/\/.*\.(link|xyz|tk|ml|ga|cf|gq|pw|top|club|mw|bd|ke|am|sbs|date|quest|cd|bid|cd|ws|icu|cam|uno|email|stream))$/.test(node.value)) {
```

Comment:
If I understand correctly, on guarddog they also consider `(http[s]?:\/\/bit\.ly.*)$` as a shady link, shouldn't we add it?

Reply:
I think we should ;) It's done